tests/functional/adapter/test_docs_gen_references.py

```diff
@@ -0,0 +1,117 @@
+import pytest
+from dbt.tests.adapter.basic.test_docs_generate import (
+    BaseReferences, ref_models__ephemeral_copy_sql, ref_models__docs_md
```

Comment:
It looks like this is named `BaseDocsGenReferences` [in `dbt-tests-adapter`](https://github.com/dbt-labs/dbt-core/blob/189c06dbb16f572d17337a1973aa6768d9abe8cf/tests/adapter/dbt/tests/adapter/basic/test_docs_generate.py#L459)
```suggestion
    BaseDocsGenReferences, ref_models__ephemeral_copy_sql, ref_models__docs_md
```